packages/ts-fdc3-oms/acme-service/dist/acme-service-0.1.0.js

```diff
@@ -1,3 +1,1485 @@
+(function webpackUniversalModuleDefinition(root, factory) {
+	if(typeof exports === 'object' && typeof module === 'object')
```

Comment:
Did you just format the files in dist? I don't think they should be changed manually - they would change themselves after a build is triggered